newinstaller/style.css

```diff
@@ -244,3 +253,32 @@ input[type=checkbox] {
 .backButton:active {
     box-shadow: -2px 2px 2px var(--text-dark);
 }
+
+.progressBar{
+    color: var(--text-dark);
+    font-family: var(--body-font);
+    font-style: italic;
+    font-size: 14px;
+    max-width: 60%;
+
+    .progressBarLabels{
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: .25rem;
+    }
+
+    .progressBarBorder{
+        display: flex;
+        justify-content: space-between;
+        /* border: 1px solid var(--text-dark); */
+        background-color: var(--accent-color);
+        height: .75rem;
+    }
+
+    .progressBarIndicator{
+        font-style: italic;
+        background-color: var(--secondary-color);
+        width: 15%;
+
+    }
+}
```

Comment:
These selectors are nestled, is this SASS or is it a mistake? Rejecting it for now.

Reply:
Seems like CSS and not SASS, please fix.